101-DDoS-Attack-Prevention/README.md

```diff
@@ -0,0 +1,189 @@
+﻿# DDoS Protection attack on a Virtual Machine Scenario 
+This repository contains DDoS attack detection on a Virtual Machine with public IP <p></p>
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-DDoS-Attack-Prevention%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/> 
+</a>
+
+
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-DDoS-Attack-Prevention%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/> 
+</a>
+
+# Table of Contents
+1. [Objectives](#objectives)
+2. [Overview](#overview)
+3. [Pre-requisites](#prerequisites)
+4. [Deploy](#deploy)
+5. [Perform Attack](#attack)
+6. [Detect and Mitigate Attack](#detect)
+7. [References](#references)
+8. [Configuration validation](#config)
+
+<a name="objectives"></a>
+# Objective of the POC  
+Showcase DDoS Protection Standard on Azure resources with public IP
+
+# Overview
+It showcases following use cases
+1. Perform DDoS attack on resources in a virtual network including public IP addresses associated with virtual machines by following configuration --> DDoS Protection Standard detects attack and mitigate the DDoS attack and send alert.
+    * Virtual Network (VNet enabled DDoS Protection Standard)
+
+<a name="important-notes"></a>
+
+# Important Notes
+DDoS Protection Standard protects resources in a virtual network including public IP addresses associated with virtual machines, load balancers, and application gateways. When coupled with the Application Gateway web application firewall, DDoS Protection Standard can provide full layer 3 to layer 7 mitigation capability.  
+Refer [Azure DDoS Protection Standard](https://docs.microsoft.com/en-us/azure/virtual-network/ddos-protection-overview) for more details.
+
+
+
+<a name="prerequisites"></a>
+
+# Prerequisites
+Access to Azure subscription to deploy following resources
+
+1.  Virtual Machine with Virtual Network
+2.  OMS (Monitoring)
+
+<a name="deploy"></a>
+
+# Deploy 
+1. Deploy using "Deploy to Azure" button at the top 
```

Comment:
ok starting to see the pattern here and with the other samples - why aren't we doing these steps within the template, having them be run manually isn't really necessary and prone to failure...  Best experience here is to just do it for the users.

Reply:
this script mentioned here is used to create the metric rule to send an email upon a DDoS Attack.